src/control/cmd/dmg/pool.go

```diff
@@ -68,19 +68,15 @@ type PoolCreateCmd struct {
 	} `positional-args:"yes"`
 }
 
+var allFlagPattern = regexp.MustCompile(`^\s*(\d{1,3})\s*%\s*$`)
```

Comment:
does this really require a regex? if so then could we have a comment to make it clear why and what it is for

Reply:
Following tasks will be done:
- [ ] Add meaningful comment

From my side, I would like to keep a regex as it allows to clearly define the supported syntax.